exercises/pythagorean-triplet/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+pub fn find() -> Option<u32> {
+

```

Comment:
there should be `unimplemented!()` here, otherwise the stub will not compile because the compiler thinks `()` is being returned

The reason for this request is so that `cargo test -- --no-run` can be run on this stub, for the purpose of automatically ensuring that the signature provided in the stub is correct (I am unable to manually ensure this, because of time).